api-spec/protobuf/ark/v1/service.proto

```diff
@@ -120,17 +120,22 @@ message GetBoardingAddressResponse {
 
 message RegisterInputsForNextRoundRequest {
   repeated Input inputs = 1;
-  optional string ephemeral_pubkey = 2;
-  repeated string notes = 3;
+  repeated string notes = 2;
 }
 message RegisterInputsForNextRoundResponse {
   string request_id = 1;
 }
 
+message Musig2 {
+  repeated string cosigners_public_keys = 1;
+  uint32 signing_type = 2;
```

Comment:
Since we don't really plan to add other signing type, whay if we make it a bool field that self describes the sign ALL option?

```suggestion
  bool signing_all = 2;
```